lib/ts/axios.ts

```diff
@@ -430,27 +450,7 @@ export default class AuthHttpRequest {
                         response.status,
                         response.headers["front-token"]
                     );
-
-                    if (response.status === AuthHttpRequestFetch.config.sessionExpiredStatusCode) {
-                        logDebugMessage("doRequest: Status code is: " + response.status);
-                        const refreshResult = await onUnauthorisedResponse(preRequestLSS);
-                        if (refreshResult.result !== "RETRY") {
-                            logDebugMessage("doRequest: Not retrying original request");
-                            // Returning refreshResult.error as an Axios Error if we attempted a refresh
-                            // Returning the response to the original response as an error if we did not attempt refreshing
-                            returnObj = refreshResult.error
-                                ? await createAxiosErrorFromFetchResp(refreshResult.error)
-                                : await createAxiosErrorFromAxiosResp(response);
-                            break;
-                        }
-                        logDebugMessage("doRequest: Retrying original request");
-                    } else {
-                        if (response.status === AuthHttpRequestFetch.config.invalidClaimStatusCode) {
-                            await onInvalidClaimResponse(response);
-                        }
-
-                        return response;
-                    }
+                    return response;
```

Comment:
Please add a comment here explaining why we do not need to check the status here.

Reply:
done.